db/migrate/20220428214730_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
Hi @rukundoeric , thank you for your review, concerning the changes you requested, I encountered that problem while generating the table, user_id is the possible solution, if you try migrating the files yourself, you will see that since there is no Author table, rails won't generate an author_id, so if you have a solution you please let me know. because everyone is generating the table with user_id not author_id. thanks.

Reply:
This is how you do it. 
```rb
t.references :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index (but students can also add column and index in separate commands)